symengine/polynomial.h

```diff
@@ -244,17 +244,44 @@ class UnivariateExprPolynomial
     {
     }
     UnivariateExprPolynomial(int i)
-        : poly_(UnivariatePolynomial::create(symbol(""), {{0, Expression(i)}}))
+        : poly_(UnivariatePolynomial::create(symbol(""), {{Expression(i)}}))
```

Comment:
Because `UnivariatePolynomial::create` uses `from_vec` to create the polynomial. The former code would have thrown an is_canonical error